assets/README.md

```diff
@@ -1,58 +1,154 @@
-
-## Using Mayflower-Tokens in Your Project
-1. Install mayflower-tokens into your project as a dependency.
-`npm i @massds/mayflower-tokens --save`
-2. Import color tokens into your SASS/SCSS file
-`@import '[path to node_modules]/@massds/mayflower-tokens/colors/mayflower-color_tokens';`
-3. Refer to variables in `@massds/mayflower-tokens/colors/_mayflower-color_tokens.scss` or a [live demo for the colors](https://mayflower.digital.mass.gov/react/?path=/story/brand-colors--colors) to map the Mayflower color tokens in your theming SCSS.
-4. You can also find the web fonts Texta that are used in Mayflower (please)
-To import into your styles, refer to [mayflower Texta font imports](https://github.com/massgov/mayflower/blob/develop/assets/scss/00-base/_fonts.scss).
+# Mayflower Assets
+The styles provided by this package require compiling with a tool that supports the [npm sass package](https://www.npmjs.com/package/sass). [Bourbon](https://www.npmjs.com/package/bourbon) 5.x is required to use the .scss files in this package and must be installed by the user of it.
 
 ## License
 Please note that the fonts and the svg-icons are licensed only for the usage on websites that are **owned by the Commonwealth of Massachusetts**. Mayflower is currently using a licensed web font Texta and purchased [Smashing Icons](https://smashicons.com/) for some icons.
 
-### Texta License
+## Texta License
 Fontspring grants Licensee a perpetual, worldwide, non-exclusive and non-transferrable license to install the Texta as webfont on Websites that are owned by the Commonwealth of Massachusetts using the @font-face selector in CSS files. For other usage, you may have to acquire an additional license through Fontsprint.
 For more details about Texta's terms of use, please refer to our [web font license](https://www.fontspring.com/lic/htswufoczd).
 
-## Publish Package
+## Publishing Instructions
+>To publish the package, you will have to be a collaborator or have access to the [@massds/assets npm package](https://www.npmjs.com/package/@massds/assets).
 
-1. `npm login` — Login to the npm account.
->To publish the package, you will have to be a collaborator or have access to the [@massds/mayflower-tokens npm package](https://www.npmjs.com/package/@massds/mayflower-tokens).
+1. Once you're done making changes and ready to publish, run `npm run build`. The provided gulp file will copy over all fonts, images, and scss files to the `dist/` directory for you.
+2. `npm login` — Login to the npm account.
+3. Update `version` in package.json and run `npm publish`. Wait a few minutes and check [@massds/assets](https://www.npmjs.com/package/@massds/assets) on the npm registry.
 
-1. `npm run build` — Build the distribution folder `dist`.
 
-1. Update `version` in package.json.
-1. `npm publish` — Publish `@massds/mayflower-tokens` to npm registry.å
-1. Wait a few minutes and check [@massds/mayflower-tokens][npm] on the registry.
 
+## Setup and Usage (fonts/images)
```

Comment:
So I think there are three use cases for this package:
1. fonts images and colors variables
2. SCSS
3. compiled CSS (create an index.scss and compile that file using example gulp setup)

I think you have most of the things documented. Can we organize the documentation to support these use cases?

Reply:
follow up in documentation improvement